code/game/gamemodes/cult/runes.dm

```diff
@@ -1,45 +1,134 @@
 var/list/sacrificed = list()
 
-/obj/effect/rune
+/*
 
+Runes are used by the cult to cause many different effects and are paramount to their success.
+They are drawn with an arcane tome in blood, and are distinguishable to cultists and normal crew by examining.
+Fake runes can be drawn in crayon to fool people.
+Runes can either be invoked by one's self or with many different cultists. Each rune has a specific incantation that the cultists will say when invoking it.
 
-///Teleport: Teleports the user to a random teleport rune with the same rune. Can be imbued into a talisman.
+TO ADD A NEW RUNE:
 
+1. Define a new subtype where the others are stored, including the invocation procs.
+2. Add an entry to the switch in determine_icon().
 
-/obj/effect/rune/proc/teleport(key)
-	var/mob/living/user = usr
-	var/allrunesloc[]
-	allrunesloc = new/list()
-	var/index = 0
-	//	var/tempnum = 0
-	for(var/obj/effect/rune/R in world)
-		if(R == src)
-			continue
-		if(R.word1 == wordtravel && R.word2 == wordself && R.word3 == key && R.z != ZLEVEL_CENTCOM)
-			index++
-			allrunesloc.len = index
-			allrunesloc[index] = R.loc
-	if(index >= 5)
-		user << "<span class='danger'>You feel pain, as rune disappears in reality shift caused by too much wear of space-time fabric</span>"
-		if (istype(user, /mob/living))
-			user.take_overall_damage(5, 0)
+*/
+
+/obj/effect/rune
+	name = "rune"
+	desc = "A strange collection of symbols drawn in blood."
+	var/cultist_desc = "An malformed, improper rune. Invoking this may cause problems." //This is shown to cultists who examine the rune in order to determine its true purpose.
+	anchored = 1
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	unacidable = 1
+	layer = TURF_LAYER
+
+	var/invocation = "Aiy ele-mayo!" //This is said by cultists when the rune is invoked.
+	var/type = "invalid" //This is used to determine the icon and color of the rune after it is drawn.

```

Comment:
afaik this will prevent it from compiling due to the existence of the type var already (the one every datum has)